repos/spack_repo/builtin/packages/rivet/package.py

```diff
@@ -154,6 +158,14 @@ def configure_args(self):
             args += ["--with-hepmc3=" + self.spec["hepmc3"].prefix]
             args += ["--with-hepmc3-libpath=" + self.spec["hepmc3"].libs.directories[0]]
 
+        val = self.spec.variants["plugin-match"].value
+        if val != "none":
```

Comment:
```suggestion
        if val != []:
```
?

Reply:
I think they are tuples, but I'm not sure.